packages/mobile_app/lib/common/ambiance/ambiance_subject_item.dart

```diff
@@ -0,0 +1,128 @@
+import 'common.dart';
+
+class AmbiacneSubject extends StatelessWidget {
+  final Function onOptionsTap;
+  final UserEntity subject;
+  final double compatibility;
+  const AmbiacneSubject({
+    Key key,
+    @required this.onOptionsTap,
+    @required this.subject,
+    @required this.compatibility,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final sign = subject.model.birth.astroSign;
+    final birthDate = subject.model.birth.toDateTime;
+    final isNegative = (compatibility - 50.0) < 0.0;
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        /// {NAME} {ROLE}
+        Padding(
+          padding: const EdgeInsets.only(
+            top: 16.0,
+            left: 16.0,
+          ),
+          child: Row(
+            children: [
+              Text(
+                "${localeText.you.capitalize()}",
+                style: AppTextStyle.userName,
+              ),
+              const Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 8.0),
+                child: const Icon(
+                  Icons.compare_arrows,
+                  color: Colors.white,
+                  size: 32.0,
+                ),
+              ),
+              SingleChildScrollView(
+                scrollDirection: Axis.horizontal,
+                child: Text(
+                  "${subject.model.name.capitalize()} (${subject.role.capitalize()})",
+                  style: AppTextStyle.userName,
+                ),
+              ),
+            ],
+          ),
+        ),
+
+        /// {Astrosign} {Birthdate}
+        Padding(
+          padding: const EdgeInsets.symmetric(
+            horizontal: 16.0,
+          ),
+          child: SizedBox(
+            height: 32,
+            child: Row(
+              children: <Widget>[
+                SvgPicture.asset("assets/icons/$sign.svg"),
```

Comment:
Optional: Btw, why to use both svg loaders? Isn't a time to decide?

Reply:
Some SVG were optimized for flutter_svg, others aren't.